builders/buildPost.ts

```diff
@@ -1,4 +1,4 @@
-import { Post } from "../models";
+import { Post, PostLink } from "../models";
```

Comment:
What do you think about naming it `SourceLink` or `FacebookLink` or similar to more clearly indicate that it's facebook link (and not calmbook interrnal post link).

Reply:
Lets put this as part of post scrapper creation task